reactor-netty-http/src/main/java/reactor/netty/http/client/Http2Pool.java

```diff
@@ -493,6 +505,30 @@ void recordInteractionTimestamp() {
 		this.lastInteractionTimestamp = clock.millis();
 	}
 
+	@Nullable
+	Borrower pollPending(ConcurrentLinkedDeque<Borrower> borrowers, boolean pollFirst) {
+		Borrower borrower = pollFirst ? borrowers.pollFirst() : borrowers.pollLast();
+		if (borrower != null) {
+			PENDING_SIZE.decrementAndGet(this);
+		}
+		return borrower;
+	}
+
+	void removePending(ConcurrentLinkedDeque<Borrower> borrowers, Borrower borrower) {
+		if (borrowers.remove(borrower)) {
+			PENDING_SIZE.decrementAndGet(this);
+		}
+	}
+
+	int addPending(ConcurrentLinkedDeque<Borrower> borrowers, Borrower borrower, boolean first) {
+		if (first) {
+			borrowers.offerFirst(borrower);
+		} else {
```

Comment:
checkstyle: else on a new row

```suggestion
		}
		else {
```